app/views/users/_profile.html.erb

```diff
@@ -77,6 +77,8 @@
     </div>
   <% end %>
 
+  <%= render partial: "users/integrations_github_repositories", locals: { show_integration: @github_repositories_show } %>
```

Comment:
@lisasy  It's quite weird we show github repos here - shouldn't that be part of "Extensions"?

Reply:
Initially, this has always been apart of the "Integrations" tab, but hear me out: as I was editing the title + subtitle of the this card, I realized that the goal of this setting is so that users who have linked GitHub to their Forem account can _pin their repositories to their profile page_. That's why I wanted to put it in "Profile" because that's what this setting impacts.

This is at the same time a type of "integration", yes, but the value it provides to people might be more relevant in Profile. I could imagine in the future, there might be similar "integrations" like this... What do you think?

Reply:
I'm not sure TBH, mostly because as you know there's a plan to split profile into its own sections (tabs) - remember that design where we had "Overview" as a profile view - it would include all the profile details as well as "entry points" for all kinds of subpages for profile - like `https://dev.to/lisasy/posts` or `https://dev.to/lisasy/comments` or `https://dev.to/lisasy/gallery` (if we have extension for galleries one day) or `https://dev.to/lisasy/repos` and so on. I imagine "GitHub Repos" will definitely  be displayed on the profile overview but only as an entry point to `https://dev.to/lisasy/gtihub-repos` subpage if you know what I mean. Therefore I think it's "settings" should be placed within "Extensions" tab, not the "Profile". 

I do understand the idea of _extensions extending the profile view_, but I still feel like they should be contained within "Extensions" page at least until we have better understanding and guidelines around WHAT can be placed WHERE in the overall IA. 

Would you agree with this sentiment? Does that make sense?

Reply:
@ludwiczakpawel That totally makes sense -- thanks for reminding me of that approach we want to move towards.

I'll move the Github repos back to extensions and make a commit!